components/src/preact/textInput/text-input.tsx

```diff
@@ -111,7 +121,11 @@ const TextInputInner: FunctionComponent<TextInputInnerProps> = ({ lapisField, pl
             </div>
             <datalist id={lapisField}>
                 {data.map((item) => (
-                    <option value={item} key={item} />
+                    <option
+                        value={item[lapisField]}
+                        key={item[lapisField]}
+                        style='white-space: nowrap;'
+                    >{`${item[lapisField]} (${item['count']})`}</option>
```

Comment:
You don't need a template string here. You can use JSX string rendering directly.
```suggestion
                    >{item[lapisField]} ({item['count']})</option>
```